Lombiq.NodeJs.Extensions/scripts/markdown-to-msbuild.js

```diff
@@ -0,0 +1,123 @@
+const fs = require('fs');
+const markdownlint = require('markdownlint'); // eslint-disable-line import/no-unresolved -- False positive, it's in the package.json.
+const path = require('path');
+const process = require('process');
+const textlint = require('textlint'); // eslint-disable-line import/no-unresolved -- False positive, it's in the package.json.
+const findRecursively = require('./find-recursively');
+
+const markdownlintConfig = JSON.parse(fs.readFileSync(path.resolve(__dirname, '..', 'config', 'lombiq.markdownlint.json')));
+
+const textLintConfig = {
+    exclude: [
+        // License files are full of legalese, which can't and shouldn't be analyzed with tools made for normal prose.
+        'License.md',
+        // The wwwwroot directory contains built and vendor assets. Any Markdown file there is not our responsibility.
+        'wwwroot',
+    ],
+    rules: [
+        'common-misspellings',
+        'doubled-spaces',
+        // "no-dead-link", // Disabled because it can't ignore relative links and can't reliably verify GitHub URLs.
+        'no-todo',
+        'no-zero-width-spaces',
+        'no-start-duplicated-conjunction',
+        'max-comma',
+        'no-empty-section',
+    ],
+};
+
+function mergeConfigs(baseConfiguration, rcFileName) {
+    const customOptions = fs.existsSync(rcFileName) ? JSON.parse(fs.readFileSync(rcFileName, 'utf8')) : {};
+
+    return {
+        ...baseConfiguration,
+        ...customOptions,
+    };
+}
+
+function getMarkdownPaths() {
+    let rootDirectory = process.argv.length > 2 ? process.argv[2] : '.';
+
+    // Traverse up the path until a .NET solution file (sln) is found.
+    if (rootDirectory === '_solution_') {
+        rootDirectory = path.resolve('.');
+        while (!fs.readdirSync(rootDirectory).some((name) => name.endsWith('.sln'))) {
+            rootDirectory = path.resolve(rootDirectory, '..');
+        }
+    }
+
+    return findRecursively(rootDirectory, [/\.md$/i], [/^node_modules$/, /^\.git$/, /^obj$/, /^bin$/]);
```

Comment:
I don't want to exclude all dot directories. Maybe the user wants to put the GHA-related documentation in _.github/docs_ or something. I exclude the ones listed, because they are managed by someone or something else (other developers, git, and dotnet respectively).

I agree about wwwroot. Actually it doesn't need to be in `textLintConfig`, just here.

Reply:
Fair enough. I was specifically thinking of `.vs`, but `.vscode` and `.idea` could also be in the list, I guess.